tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+import copy
+import json
+
+import ocdskit.oc4ids_transforms as transforms
+from tests import read
+
+
+def test_initial_tranform_state():
+    releases = json.loads(read("release-package_additional-contact-points.json"))["releases"]
+    transform_state = transforms.InitialTransformState(releases, "1")
+    assert len(transform_state.compiled_releases) == 1
+    assert len(transform_state.releases_by_ocid["ocds-213czf-1"]) == 2
+
+
+def test_run_all():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["publicAuthority"]}],
+        }
+    ]
+    output = transforms.run_transforms({}, releases, "1")
+    assert output["parties"] == releases[0]["parties"]
+
+
+def test_public_authority_role():
```

Comment:
Should this test also cover the scenario where there are multiple contracting processes, each with a different publicAuthority (or buyer) but with the same value in `parties/id`?